docs/introduction.rst

```diff
@@ -1,73 +1,47 @@
 
-|AppName| is intended to provide an extensible platform to acquire scientific data from a variety of microscopes and other instruments, be intuitive and easy to use, and allow sophisticated data visualization, processing, and analysis.
+Click to download Nion Swift or for :ref:`installation` instructions.
 
-Key Features
+Click here to get started fast with our :ref:`basic-use`.
 
-- Data handling of 1D plots, 2D images, 1D and 2D collections of plots and images, and sequences.
-- Live computations that can be chained during acquisition or live parameter adjustment.
-- Open source, cross platform (macOS, Windows, Linux), and Python based scientific data processing.
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is comprised of a central section called the workspace and the surrounding area for utility panels. 
 
-Data Items and Display Items
-----------------------------
-|AppName| operates on data arrays, which include 2D images and 1D spectra, which are generally called *Data Items*.
+The central workspace is for display panels which can either display a data item or can display thumbnail images of multiple data items side by side in one of the browser formats. For more information, see :ref:`display-panels`.
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The utility panels surrounding the central workspace allow you to make adjustments to a selected data item in a display panel. For more information on their specific functions, see :ref:`user-interface`.
 
-A spectrum image would be a 2D collection of 1D data. This would be typical data acquired from a STEM microscope spectrum imaging acquisition.
+Data and Display
+================
+Data is stored in a project as data items. Data items can be dragged into display panels to be displayed. Data items that are displayed in a display panel can be manipulated with graphics, computations, and more. These displayed data items are referred to as display items. Display items can be modified and viewed to see various aspects or values of the original data item. 
 
-Data Items are displayed using one or more *Display Items*.
+Read more in :ref:`data-items`.
 
-See :ref:`data-items`.
+Graphics and Masks
+==================
+Nion Swift provides plenty of useful graphics to narrow down your view of the data. Use graphics to highlight portions of the data, select areas for processing, or to focus on specific sections of data.
```

Comment:
"Nion Swift provides graphics to annotation and select parts of your data." (n.b. "plenty of useful" doesn't add much and every prepositional clause makes it more difficult for non-native-English speakers).

You could also combine both sentences here since they say the same thing. I like the second sentence.